src/dw_design_system/dwds/components/spotlight.scss

```
@@ -7,4 +7,8 @@
     --content-main-padding: 0;
     --content-main-bg-color: transparent;
   }
+
+  h3 {
```

Let's target only the h3 at this level and not all h3's inside this element

```suggestion
  > h3 {
```